app/Console/Commands/SendToQuasar.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Post;
+use Rogue\Models\Signup;
+use Illuminate\Console\Command;
+use Rogue\Jobs\SendPostToQuasar;
+use Rogue\Jobs\SendSignupToQuasar;
+
+class SendToQuasar extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:quasar
+                            {start=0000-00-00 : Include all signups and posts updated on or after this day. Format: YYYY-MM-DD}
```

Comment:
lol is the default the beginning of time?

Reply:
haha yeah to grab all of them! I started out by checking if a `start` was provided or not, but it seemed like this way was going to be less code and easier to follow.

Reply:
ah I see now!